modules/ossm-release-notes-3-0-5.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/ossm-release-notes/ossm-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-release-3-0-5_{context}"]
+= {SMProductName} version 3.0.5
+
+This release of {SMProductName} is included with the {SMProductName} Operator 3.0.5 and is supported on {ocp-product-title} 4.16 and later. This release addresses enhancements and Common Vulnerabilities and Exposures (CVEs). For supported component versions for 3.0.5, see "Service Mesh version support tables".
+
+[id="ossm-enhancements-3-0-5_{context}"]
+== Enhancements
+
+* This enhancement updates {istio} to version 1.24.6.
+
+* This enhancement updates Kiali operator to version 2.4.9.
```

Comment:
this is not true. we don't release kiali operator for each version of OSSM. Only the latest kiali operator one is released ( 2.11.3 ) which is able to managed more control plane versions ( for 3.0.5, it is kiali v2.4, which is 2.4.9 )